app/lib/forms/qualified_teacher_check.rb

```diff
@@ -120,12 +113,33 @@ def active_alert?
       @active_alert
     end
 
+    def before_render
+      # Load info from current_user into store if it isn't already set
+      # This way this info will be pre-populated in the forms.
+      # This is useful for when the user is logged in and we already know this info.
+      wizard.store["full_name"] ||= query_store.current_user.full_name
+      wizard.store["trn"] ||= query_store.current_user.trn
+      wizard.store["date_of_birth"] ||= query_store.current_user.date_of_birth
+    end
+
     def after_save
       wizard.store["trn_verified"] = trn_verified?
       wizard.store["trn_lookup_status"] = trn_lookup_status
       wizard.store["trn_auto_verified"] = trn_auto_verified?
       wizard.store["verified_trn"] = verified_trn
       wizard.store["active_alert"] = active_alert?
+
+      query_store.current_user.update!(
+        trn: trn_to_store,
+        full_name:,
+        date_of_birth:,
+        national_insurance_number: ni_number_to_store,
+        trn_verified: trn_verified?,
+        trn_lookup_status:,
+        trn_auto_verified: !!trn_auto_verified?,
+        active_alert: active_alert?,
+      )
+      wizard.store["trn_set_via_fallback_verification_question"] = true
```

Comment:
This controls whether to show the name/email/nino/trn fields on the check_answers page, since we need to display those when the user has to fallback to this page.

Reply:
It's also useful to just have the flag stored so we can identify these users.